vendor/github.com/rubiojr/go-vhd/vhd/vhd.go

```diff
@@ -324,8 +327,14 @@ func (vhd *VHD) PrintExtraHeader() {
 	fmtField("Parent timestamp", fmt.Sprintf("%s", t))
 
 	fmtField("Reserved", hexs(header.Reserved[:]))
-	parentName := utf16BytesToString(header.ParentUnicodeName[:],
-		binary.BigEndian)
+	parentNameBytes, _, err := transform.Bytes(
+		unicode.UTF16(unicode.BigEndian, unicode.IgnoreBOM).NewDecoder(),
```

Comment:
looks like this only handles BigEndian... whereas utf16BytesToString claimed to handle big or little endian... is BigEndian always correct here?

Reply:
cc @rubiojr

Reply:
Yes, only BigEndian is required, according to the VHD specs. Even if the function that was removed handled both, it was hardcoded to BigEndian previously also, so no behavior change here.